interfaces/cython/cantera/reaction.pyx

```diff
@@ -1202,8 +1202,8 @@ cdef class Reaction:
 
         .. deprecated:: 2.6
              This property is for temporary backwards-compatibility with the deprecated
-             `PlogReaction` class. Replaced by `Reaction.rate.rates` for reactions where
-             the rate is a `PlogRate`.
+             `PlogReaction` class. Replaced by ``Reaction.rate.rates`` for reactions
```

Comment:
It wasn't doing it for me when I was testing locally. I don't know how to get Sphinx to link directly to member variables/functions of a different class (and in this case, I think it would have trouble knowing the type of `Reaction.rate` to be able to figure out what class the `rates` property was a member of).

Reply:
That's fine then ☺️